ml/detections.go

```diff
@@ -21,7 +21,7 @@ const (
 
 // FormatDetectionOutputs formats the output tensors from a model into detections.
 func FormatDetectionOutputs(outNameMap *sync.Map, outMap Tensors, origW, origH int,
-	boxOrder []int, labels []string,
+	boxOrder []int, labels []string, wasResized bool, resizedW, resizedH int,
```

Comment:
You cannot change the signature of this public function - I believe it used by ML @tahiyasalam